build-logic/buildquality/src/main/kotlin/gradlebuild.ci-reporting.gradle.kts

```
@@ -96,13 +93,13 @@ fun verifyTestFilesCleanup(failedTasks: List<Task>, tmpTestFiles: List<File>) {
     }
 }
 
-fun prepareReportsForCiPublishing(failedTasks: List<Task>, executedTasks: List<Task>, tmpTestFiles: List<File>) {
-    val failedTaskCustomReports = failedTasks.flatMap { it.failedTaskGenericHtmlReports() }
+fun prepareReportsForCiPublishing(tasksToCollectReports: List<Task>, executedTasks: List<Task>, tmpTestFiles: Collection<File>) {
+    val collectedTaskCustomReports = tasksToCollectReports.flatMap { it.failedTaskGenericHtmlReports() }
```

```suggestion
    val collectedTaskHtmlReports = tasksToCollectReports.flatMap { it.genericHtmlReports() }
```

I don't know why this was called `custom` reports, though these seem to be the generic html reports.